intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -386,8 +386,12 @@ question_values %in% values_1_to_20
 _Data frames are the most fundamental unit of data analysis in R._ 
 They are tables which consist of rows and columns, much like a _spreadsheet_. 
 Each column is a variable which behaves as a _vector_, and each row is an observation. 
-We will begin our exploration with the old trusted dataset `iris`, which comes with R. 
-Learn about this dataset using the standard help approach of `?iris`.
+We will begin our exploration with dataset about penguins from the `palmerpenguins` package. 
```

Comment:
I think we should have a little bit of explanation of the dataset here. At least a link to https://allisonhorst.github.io/palmerpenguins/ and maybe even a picture?

![Penguins](https://allisonhorst.github.io/palmerpenguins/reference/figures/lter_penguins.png) Artwork by @allison_horst.